ush/parsing_model_configure_FV3.sh

```diff
@@ -31,7 +31,11 @@ local WRITE_GROUP=${WRITE_GROUP:-1}
 local WRTTASK_PER_GROUP=${WRTTASK_PER_GROUP:-24}
 local ITASKS=1
 local OUTPUT_HISTORY=${OUTPUT_HISTORY:-".true."}
-local HISTORY_FILE_ON_NATIVE_GRID=".true."
+if [[ "${DO_JEDIATMVAR:-}" == "YES" ]]; then
```

Comment:
Should this be set in a config file, so that these lines in ```parsing_model_configure_FV3.sh``` become:

```local HISTORY_FILE_ON_NATIVE_GRID=${history_file_on_native_grid:-".false."}```

and ```history_file_on_native_grid``` is set in ```config.ufs```